### DISCUSSION TITLE
Keystroke Overlay

### DISCUSSION THREAD

**Discussion by knkrth** [shhdharmen/keyboard-css#8] [Ideas]
This is not related to the [keyboard-css](https://github.com/shhdharmen/keyboard-css) 

Recent days everyone turn's to visual tutorials & there are only few tools are out there to support them.

One of them is displaying shortcut keystroke on their videos, There are tools out there but they are expansive. 

Is it possible to make a keystroke capture similar to https://ixeau.com/keystroke-pro/ on browser?

These are the reference so far I find on github,

- [analog_keyboard_overlay](https://github.com/DarrenVs/analog_keyboard_overlay )
- [keystroke](https://github.com/oliviernt/keystroke)

**Comment by shhdharmen** [comment]
Interesting idea!

But when you said "make it for browser", you mean making an extension?

**Reply by knkrth** [reply]
Nope, just a website which displays the keystroke.

**Reply by shhdharmen** [reply]
OK. You can easily create such website and use keyboard-css in it.

**Reply by knkrth** [reply]
Ok, Thanks.
